src/WooCommerce.php

```diff
@@ -1492,4 +1492,32 @@ private static function getRepricingOptionsDropdownValues(\WC_Product $product)
     return $dropdown_values;
   }
 
+  /**
+   * Adds custom meta to customer.
+   *
+   * @implements woocommerce_checkout_create_order
+   */
+  public static function addCustomMetaForUser($order) {
+    $user_id = $order->get_user_id();
+
+    $args = [
+      'limit' => -1,
+      'customer_id' => $user_id,
+    ];
+    $customer_orders = wc_get_orders($args);
+
+    $items_counter = $order->get_item_count();
```

Comment:
If you initialise `$customer_orders` you don't to check before updating meta:
```suggestion
    $customer_orders = 1;
    $items_counter = $order->get_item_count();
```

Reply:
I can't do this because the woocommerce_checkout_create_order hook is triggered before the save of the current order. So the current order doesn't appear in the query. Thats why we need to get the item count and add a +1 to customer_orders_count field.